csharp/src/IceDiscovery/Locator.cs

```diff
@@ -65,59 +89,12 @@ public ValueTask SetAdapterDirectProxyAsync(string adapterId, IObjectPrx? proxy,
             return new ValueTask(Task.CompletedTask);
```

Comment:
I think this should just be `new ValueTask()` or we could declare the method with "async"?